### ISSUE TITLE
Yandex and meta fragment

### ISSUE THREAD

**Issue by tuhoojabotti** [prerender/prerender#204]
Yandex documentation states that you should omit the fragment meta tag from the static version:

> You should not place the meta tag in the HTML version of the document. If you do so, the page will not be indexed.

https://help.yandex.com/webmaster/robot-workings/ajax-indexing.xml

Might be a good idea to make a plugin to remove the meta tag from the html, what do you think?

**Comment by thoop** [comment]
I interpret that as, "If you only place the meta tag in the HTML version of the document, the page will not be indexed"....meaning that you need to place the meta tag in the AJAX version of the document so that they request the HTML version.

We have the meta tag in the HTML version of https://prerender.io/ and yandex still indexes our page just fine.

That seems to invalidate their statement if you interpret it literally, but I'm open to discussing it more if you find some more information on that.

**Comment by tuhoojabotti** [comment]
Could be a result of bad English. My client asked about bad ranking on Yandex (and other search engines) after we released their new site with Angular and Prerender. I found that page and decided to make an issue for discussion, but I think I'll have to agree with you. I think the bigger problem is that all their old links are now dead and it takes time to gain the same ranking on search engines.
